wiki/impaled.md

```diff
@@ -5,31 +5,100 @@ slug: impaled
-  - Only adds 0.75 heart of damage per level instead of 1.25.
-  - For the Hellfork, Impaling will do one extra heart of damage per  level to all fire immune entities (entities with Fire Resistance  included!)
-- Loyal Tridents return in the right slot (including secondary hand)
-- (Creative Only) A new powerful item, the Maelstrom, that allows its user to throw tridents out of their inventory
-  - Accepts Efficiency (reduces the time between each trident launch), Unbreaking and Mending
+**Impaled** is a fabric 1.17 mod that adds a few new tridents to the game and reworks some of the mechanics evolving around them.
+
+## Wiki
+
+### Mechanics
+
+Impaled changes, fixes and adds a few features for tridents:
+
+#### Impaling
+
+Impaling now deals damage to mobs that are wet (underwater or in rain) instead of aquatic mobs. To balance this change Impaling only increases the damage by 0.75 per level instead of 1.25.
+
+#### Loyalty
+
+Tridents with the Loyalty enchantment will always return to the inventory slot they were launched from, including the Offhand. If the player picks up new items blocking said slot, the trident will return to a free slot just as usual.
+
+#### Sincere Loyalty
+
+This mod also adds a new level to Loyalty, Sincere Loyalty or Loyalty IV, giving you the ability to retain your Trident from anywhere in the world by right-clicking with an empty slot. Tridents with Sincere Loyalty will drop as entity instead of as an item in any situation and will even return to you when in another player's inventory. In addition to that, Tridents with Sincere Loyalty can be enchanted with Riptide.
+
+##### Obtaining
+
+Sincere Loyalty can be obtained by combining a Loyalty III-Trident with a Heart of the Sea in a Smithing Table.
+
+![Sincere Loyalty](impaled/SincereLoyaltyRecipe.png){: .wiki}
+
+### Tridents
+
+There are 4 new Tridents added by Impaled:
+
+#### Pitchfork
+
+The Pitchfork is an early-game version of the Trident. It has reduced durability compared to the actual Trident and does not work properly underwater. Nonetheless it can be thrown and used in melee combat, enchanted with all the Trident-Enchantments and even be used to convert dirt to farmland.
+
+![Pitchfork](impaled/Pitchfork.png){: .wiki}
+
+##### Obtaining
+
+The Pitchfork can be crafted using 3 Iron Bars and 2 Sticks:
+
+![Pitchfork Recipe](impaled/PitchforkRecipe.png){: .wiki}
+
+#### Elder Trident
+
+This Trident uses the ancient Guardian magic to home for mobs if it does not hit its target first time. It will also store the items dropped by killed mobs and give them to the player when being picked up or retrieved with Loyalty. 
+
+If a Riptide-Elder Trident is used it will create 1 Guardian Trident per Riptide level that will copy the Elder Trident's behavior of homing for mobs and returning items.
+
+A Channelling-Elder Trident will not only hit its enemies with a lightning bolt but also give them the Apparition and the Mining Fatigue Effect.
```

Comment:
I'd rather put something along the lines of:
"In addition to the normal behaviour of Channeling during storms, hitting a player with an Elder Trident will affect them like an Elder Guardian, appearing before them and affecting them with Mining Fatigue (not limited to thunderstorms)."